server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -5056,6 +5091,7 @@ public void finalizeStop(VirtualMachineProfile profile, Answer answer) {
             }
         }
 
+        updateResourceCount(owner.getAccountId(), owner.getDomainId());
```

Comment:
this might take longer time if there are many accounts.

Reply:
@weizhouapache  Do you mean we should move this to updateVM workflow and use `resourceCountIncrement` / `resourceCountIncrement` as opposed to ` updateResourceCount`?

Reply:
@Pearl1594 yes.